raiden/channel.py

```diff
@@ -410,6 +410,7 @@ def settled_block(self):
         return self._settled_block
 
     def set_opened(self, block_number):
+        # TODO: ensure the same callback logic as in set_settled
```

Comment:
Add the issue number that tracks this here: https://github.com/raiden-network/raiden/issues/393
Question: Did you encounter a callback being invoked twice?